moped-editor/src/utils/activityLogFormatters/mopedFundingActivity.js

```diff
@@ -2,6 +2,19 @@ import MonetizationOnOutlinedIcon from "@material-ui/icons/MonetizationOnOutline
 import { ProjectActivityLogTableMaps } from "../../views/projects/projectView/ProjectActivityLogTableMaps";
 import { isEqual } from "lodash";
 
+const getFundingSourceIdText = (newRecord, fundingSources, fundingPrograms) => {
```

Comment:
Sweet! Just double-checking that we would want the change to prefer source over program if they are both filled out. Maybe that doesn't really happen so disregard if so. I just filled in all fields the first time around because I didn't read the instructions well 🙃